src/CSharp/AspnetWebMvc/AspnetWebMvc/Web.config

```diff
@@ -29,10 +29,12 @@
     
     <add key="IdentifyOauth2:CodeFlowClientId" value="webmvc_codeflow_id"/>
     <add key="IdentifyOauth2:CodeFlowRedirectUri" value="http://localhost:62640/Home/CodeFlowCallback"/>
+    <add key="IdentifyOauth2:IdTokenHint" value="eyJhbGciOiJSUzI1NiIsImtpZCI6IlBCX1hOYVFEWGp0NDB6UkUzbE1udzVPcUtDNCIsIng1dCI6IlBCX1hOYVFEWGp0NDB6UkUzbE1udzVPcUtDNCIsInR5cCI6IkpXVCJ9.eyJ1aUlkIjoiZGQ0YjllMzQtOThhZC00MDk1LWIyZGQtZDFmM2Q2YjRlOWNkIiwic3ViIjoiQ2ZESjhPeVJ1OTNLUlp4Qm1Eb2Z2TjV3eGh6c2FhSEJyOU5LYS05b1dUaEh3cnZ6RTQ0ZWtSdG1aYXpRdlBnWVZvX3dnWnF6SzNQY3lnVFVjM1lJUzRjY1puNUJNYklnMHZURUtCLVllcDJFRklOX2s5bTJacnVJRklKdnhqVjZRY2gzSkU0MUY2NVFNR3RnZGNlOEV3OWd0ZGdpdmQtY2ROTDJJTHBWVHppbmNTMzEiLCJhdXRoX3RpbWUiOjE1NjA4NDQ2NTQuMCwidG9rZW5fdXNhZ2UiOiJpZGVudGl0eV90b2tlbiIsImp0aSI6ImQxNDBlYmFmLTc3M2UtNGQ1MS1hOWE0LTk4ZmQzNzc3NDdlNyIsImF1ZCI6WyJ3ZWJtdmNfY29kZWZsb3dfaWQiLCJodHRwczovL2RldjU1LnNhZmV3aGVyZS5sb2NhbC9ydW50aW1lLyJdLCJub25jZSI6IjJkNzE0Y2ZiMGQ2NjQ1NzQ5YjU3NTM5OTZjMTQ0ZmIwIiwiYXRfaGFzaCI6IlBLczZwZjZaZjlNUm96NWZiaEw2bEEiLCJhenAiOiJ3ZWJtdmNfY29kZWZsb3dfaWQiLCJpYXQiOjE1NjA4NDQ2NTgsIm5iZiI6MTU2MDg0NDY1OCwiZXhwIjoxNTk2ODQ0NjU4LCJpc3MiOiJodHRwczovL2RldjU1LnNhZmV3aGVyZS5sb2NhbC9ydW50aW1lL29hdXRoMiJ9.ZBQckQPCw-FID3mZK24HulnqiJPYdMpbC3_jphcdiSubDdRVM-k9cGWkoZHnOsDtjtD1XMANgb_2jG_kBhTqUeNk58z3arpRhzeNA9mJQAA8V4vHBb5oA5vRJ0kzc9CBm6kQbNtSfH8Ym3URMwaYScGMDSRHrxGVTwOrFqJDmVj__BOrGBZ7WOXlPshwyfK0pePdjjH4hz2XdKpwnSXgWcRIQfljrG07HkaCK9jU4PwKEDSk1Tdabz7wmwhgDvONbam1eZ9mN_7KjFxqETw7oroHwNtBp5HZFM1tNVwnAAXjtzthQGuXPkmdcinOEq3au_3shQ397GDV3pRghy_D6A"/>
```

Comment:
why does a user need to put an IdTokenHint here? in reality, how does a user acquire this token to put it in web.config?

Reply:
I considered it as a kind of testing tool only. Anyway, I will hide it for SA only and will write a full workflow with id_token_hint if it is requested.